english/lamping/lamping.md

```diff
@@ -1,2 +1,10 @@
 As you begin scrolling through your feed on IG and using your 
 super-retweeting abilities, the realities of the universe fade...
+However, you do not notice that there is a small spark of fire burning
+right beside you. As the fire increases, you begin to smell the burn 
+from deep in your sleep,
+
+Do you:
+
+wake up hurriedly and run out of the house or
+[continue to enjoy the dream you were having of your crush](english/burn-down-the-house/house-burn.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.